package.json

```diff
@@ -85,17 +85,15 @@
     "@as-integrations/express5": "1.1.2",
     "@aws-sdk/client-cloudwatch-logs": "3.696.0",
     "@aws-sdk/client-dynamodb": "3.920.0",
-    "@aws-sdk/client-dynamodb-v3": "npm:@aws-sdk/client-dynamodb@3.20.0",
     "@aws-sdk/client-kinesis": "3.920.0",
     "@aws-sdk/client-kms": "3.920.0",
     "@aws-sdk/client-lambda": "3.920.0",
     "@aws-sdk/client-s3": "3.920.0",
-    "@aws-sdk/client-s3-v3": "npm:@aws-sdk/client-s3@3.321.1",
     "@aws-sdk/client-sns": "3.920.0",
     "@aws-sdk/client-sqs": "3.920.0",
-    "@aws-sdk/client-sqs-v3": "npm:@aws-sdk/client-sqs@3.24.0",
     "@aws-sdk/client-ssm": "3.920.0",
     "@aws-sdk/lib-dynamodb": "3.920.0",
+    "@aws-sdk/middleware-retry": "3.370.0",
```

Comment:
The dependency is used in sns utils: packages/collector/test/tracing/cloud/aws-sdk/v3/sns/utils.js

And somehow these older aws depenedencies provided this dependency to us.
Now that we removed them, we have to manually install.